framework/PageInputs/PageAsyncMultiSelect.tsx

```diff
@@ -30,6 +30,7 @@ export interface PageAsyncMultiSelectProps<ValueT>
   onBrowse?: () => void;
 
   compareOptionValues?: (a: ValueT, b: ValueT) => boolean;
+  validate?: (items: ValueT[]) => Promise<string | undefined>;
```

Comment:
The `validate` prop appears to be unused in PageAsyncMultiSelect. We could probably remove it.